constants.js

```diff
@@ -12,17 +12,24 @@ const HACKER_STATUS_CONFIRMED = "Confirmed";
 const HACKER_STATUS_CANCELLED = "Cancelled";
 const HACKER_STATUS_CHECKED_IN = "Checked-in";
 const HACKER_STATUSES = [
-    HACKER_STATUS_NONE, 
+    HACKER_STATUS_NONE,
     HACKER_STATUS_APPLIED,
     HACKER_STATUS_ACCEPTED,
     HACKER_STATUS_WAITLISTED,
-    HACKER_STATUS_CONFIRMED, 
-    HACKER_STATUS_CANCELLED, 
+    HACKER_STATUS_CONFIRMED,
+    HACKER_STATUS_CANCELLED,
     HACKER_STATUS_CHECKED_IN
 ];
+
+const HACKER = "Hacker";
+const VOLUNTEER = "Volunteer";
+const STAFF = "Staff";
+const GODSTAFF = "GodStaff";
```

Comment:
Petition to change GODSTAFF to ADMIN lol

Reply:
I'm with it

Reply:
(I fully understand that I wrote GODSTAFF originally, this is 100% my fault)

Reply:
😡